addons/editor-theme3/theme3.js

```diff
@@ -88,7 +88,11 @@ function updateSettings(addon, newStyle) {
       }
       #s3devIDD > li.${prop} {
         background-color: var(--editorTheme3-${settingName}Color);
-      }`;
+      }
+      .console-variable[data-category="${prop}"] {
+        background-color: var(--editorTheme3-${settingName}Color) !important;
+      }
+      `;
```

Comment:
What does this do? I'm not sure how theme3 behaves with this new category

Reply:
It makes custom block colors work in here:

![image](https://user-images.githubusercontent.com/33787854/121953684-7b3bd400-cd23-11eb-8c3d-3bdfb99401cc.png)